modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/utils.ts

```diff
@@ -388,14 +388,3 @@ export function updatePreviousURLParam({
 
```

Comment:
Hi @rebsilva, could you check if this action is still necessary:

`window.history.pushState(null, '', newPreviousURL.toString());`

Reply:
i will take a look at it and remove if its not necessary anymore, thanks!

Reply:
i've removed it! :cherry_blossom: